lightning/src/ln/msgs.rs

```diff
@@ -144,6 +144,13 @@ pub struct FundingLocked {
 	pub(crate) next_per_commitment_point: PublicKey,
 }
 
+impl FundingLocked {
```

Comment:
Hmmmmm...I'm torn on this one, the messages are essentially simple structs with no logic aside from serialization/deserialization so I think it may make sense to just make their fields pub directly and move on. What do you think @jkczyz.

Reply:
I'd say just make them all public. No need to for an accessor if we don't need to control how they are mutated.

Reply:
so *all* the messages, not just `FundingLocked`, right?

Reply:
and if so, are you OK with `#[allow(missing_docs)]` on the file?

Reply:
I pushed an implementation of this, but happy to nuke that commit if you want another approach

Reply:
Hmm, probably just do FundingLocked for now - there's some awkwardness around the way we represent onion data internally that would be nice to clean up first. Is there a way to allow missing docs on only the structs?

Reply:
OK, pushed a commit with `FundingLocked` fields public and `#[allow(missing_docs)]` on just that one struct.  I'm not sure how much value it adds to comment each field, since the field names are pretty mnemonic, but can also put in comments instead if you prefer.